packages/ccd-js-gen/src/lib.ts

```diff
@@ -0,0 +1,252 @@
+import * as fs from 'node:fs/promises';
+import * as path from 'node:path';
+import * as tsm from 'ts-morph';
+import * as SDK from '@concordium/common-sdk';
+import { Buffer } from 'buffer/';
+
+/** Output options for the generated code */
+export type OutputOptions =
+    | 'Typescript'
+    | 'Javascript'
+    | 'TypedJavascript'
```

Comment:
Why is this a useful option? Can you document the semantics of the different options.

Reply:
I have added documentation for the type